README.md

```
@@ -126,44 +126,45 @@ This repository contains an ESPHome BLE client for interfacing with the Powerdal
+                - [ESP32-S3-N16R8](#esp32-s3-n16r8)
+    - [Activating the Integrated ESPHome Webserver/GUI](#activating-the-integrated-esphome-webservergui)
+    - [Integrating ESPHome Devices with Home Assistant](#integrating-esphome-devices-with-home-assistant)
+    - [HTTP API Integration Optional](#http-api-integration-optional)
+        - [Configuration](#configuration)
+        - [POST Events & Endpoints](#post-events--endpoints)
+        - [Example Requests](#example-requests)
+            - [POST http://your-api-server.com/charger/status](#post-httpyour-api-servercomchargerstatus)
+            - [POST http://your-api-server.com/charger/event](#post-httpyour-api-servercomchargerevent)
+            - [POST http://your-api-server.com/charger/session/minimal](#post-httpyour-api-servercomchargersessionminimal)
+        - [Notes](#notes)
+    - [MQTT Integration Optional](#mqtt-integration-optional)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)